Client/src/Pages/Settings/index.jsx

```diff
@@ -37,10 +37,30 @@ const Settings = ({ isAdmin }) => {
   const [form, setForm] = useState({
     ttl: checkTTL ? (checkTTL / SECONDS_PER_DAY).toString() : 0,
   });
+  const [version,setVersion]=useState("");
   const [errors, setErrors] = useState({});
   const dispatch = useDispatch();
   const navigate = useNavigate();
 
+  //Fetching latest release version from github
+  useEffect(() => {
+		const fetchLatestVersion = async () => {
+			try {
+				const response = await fetch(
+					"https://api.github.com/repos/bluewave-labs/bluewave-uptime/releases/latest"
+				);
+				if (!response.ok) {
+					throw new Error("Failed to fetch latest version");
+				}
+				const data = await response.json();
+				setVersion(data.tag_name); // Set the latest version number
+			} catch (error) {
+				createToast({ body: error.message || "Error fetching latest version" }); // Set error message
+			}
```

Comment:
Why don't we do this in case our network request fails:

```
let version = "unknown"
try {
   ...
   const data = await response.json()
   version = data.tag_name
} catch (error) {
    createToast(...)
} finally {
    setVersion(version)
}
```

Now we'll always have a recent value for `version`, even if it is unknown.